data/arweave/labels.json

```diff
@@ -12,5 +12,6 @@
   "25FrqEFWhO19W3vAFJIMEtMQ46XaEfoJMxPCwxzjwCw": "KYVE Near",
   "aLemOhg9OGovn-0o4cOCbueiHT9VgdYnpJpq7NgMA1A": "Verto Exchange",
   "usjm4PCxUd5mtaon7zc97-dt-3qf67yPyqgzLnLqk5A": "Verto Token",
-  "dxGmt44SZenqmHa-_IEl8AmuejgITs4pB3oe-xUR36A": "Verto Trading Post"
+  "dxGmt44SZenqmHa-_IEl8AmuejgITs4pB3oe-xUR36A": "Verto Trading Post",
+  "65cXZ7nYYJlbSJWIGaqdIpL8llR2QOCJpKoS6qjvoHc": "Another Money"
```

Comment:
remove, your app doesn't have enough usage to be labelled yet

Reply:
It's usage is giant. First token made specially for smart contracts must be labelled. It can be used for PST distribution, decentralized markets (NFT and etc.), etc.

Reply:
Not from what I see on chain. Remove.

Reply:
What do you see on chain?